docs/cloud/integrations/README.md

```diff
@@ -0,0 +1,23 @@
+# Event Store Cloud Integrations
+
+The Event Store Cloud offers integrations between internal sources such as cluster health and issue detection or notifications events and sinks which include external services such as Slack and OpsGenie.
+
+## Integration Sources
+
+"Sources" are driven by events or other mechanism inside the Event Store Cloud.
```

Comment:
```sugggestion
"Sources" are driven by events or other mechanism inside Event Store Cloud.
```

Reply:
**COMMENT:** Events here may be misleading with the business events stored inside EventStoreDB. However, I don't have a better idea so far on how to replace it.